tests/test_regression.py

```diff
@@ -51,3 +54,27 @@ class Params:
 
         unknown_author = AuthorFactory(unknown=True)
         self.assertEqual("", unknown_author.fullname)
+
+
+class NameConflictTests(unittest.TestCase):
```

Comment:
Do you mind moving this test to `tests/test_alchemy.py`? It’s more precise than `test_regression`.